web/js/components/about/about-page.js

```diff
@@ -297,7 +297,7 @@ export default function AboutPage() {
         <hr />
         <h2>License</h2>
         <p>
-          Copyright &copy; 2013 - 2020 United States Government as represented by the Administrator of the National Aeronautics and
+          Copyright &copy; 2013 - 2021 United States Government as represented by the Administrator of the National Aeronautics and
```

Comment:
We can just output the current year here and forget about ever having to update this again:

```
 Copyright &copy; 2013 - ${new Date().getFullYear()} 
```